R/as_mapper.R

```diff
@@ -107,7 +107,7 @@ plucker <- function(i, default) {
   )
 }
 
-as_predicate <- function(.fn, ..., .mapper, .allow_na = FALSE) {
+as_predicate <- function(.fn, ..., .mapper, .allow_na = FALSE, .error_call = caller_env()) {
```

Comment:
Better take `.error_arg = caller_arg(.fn)` as well, and pass to `abort()`, and use in the message.